src/mem/ruby/structures/RubyCache.py

```diff
@@ -44,6 +44,8 @@ class RubyCache(SimObject):
         "0B", "block size in bytes. 0 means default RubyBlockSize"
     )
 
+    atomicLatency = Param.Cycles(1, "Cycles for an atomic operation")
```

Comment:
It would be helpful to add a comment here that these two parameters only apply to GPU atomics.